cmd/vela-server/queue.go

```diff
@@ -18,11 +18,12 @@ func setupQueue(c *cli.Context) (queue.Service, error) {
 
 	// queue configuration
 	_setup := &queue.Setup{
-		Driver:  c.String("queue.driver"),
-		Address: c.String("queue.addr"),
-		Cluster: c.Bool("queue.cluster"),
-		Routes:  c.StringSlice("queue.routes"),
-		Timeout: c.Duration("queue.pop.timeout"),
+		Driver:                   c.String("queue.driver"),
+		Address:                  c.String("queue.addr"),
+		Cluster:                  c.Bool("queue.cluster"),
+		Routes:                   c.StringSlice("queue.routes"),
+		Timeout:                  c.Duration("queue.pop.timeout"),
+		EncodedSigningPrivateKey: c.String("queue.signing.private-key"),
```

Comment:
agreed, im bad at naming, any suggestions on replacement? maybe since we're already in the queue package...
```suggestion
		PrivateKey: c.String("queue.private-key"),
```

Reply:
I like it